src/libs/isNative.js

```diff
@@ -0,0 +1,8 @@
+import getPlatform from './getPlatform';
```

Comment:
I wonder if this could not be made similar to the setup in `useNativeDriver`, or even re-use that (perhaps change the name)

Reply:
changed to `useNativeDriver`

Reply:
I agree with @BeeMargarida , `useNativeDriver` setup is more close to App's ways of dealing with platform differences.